.github/workflows/ci.yml

```diff
@@ -211,3 +197,29 @@ jobs:
         with:
           github_token: ${{ github.token }}
           filter_mode: file
+
+      # prepare for non-natively-supported reviewdog checks
+      - name: Install reviewdog
+        run: |
+          mkdir -p $HOME/.local/bin
+          curl -sfL https://raw.githubusercontent.com/reviewdog/reviewdog/master/install.sh \
+            | sh -s -- -b $HOME/.local/bin
+
+          echo ::add-path::$HOME/.local/bin
+
+      # run black and annotate incorrect files with reviewdog
+      - name: Install black
+        run: |
+          python3 -m pip install --upgrade pip
+          python3 -m pip install black==19.10b0
+      - name: Check Python formatting
+        env:
+          REVIEWDOG_GITHUB_API_TOKEN: ${{ github.token }}
```

Comment:
does this need reviewdog rather than just using github actions error output?

Reply:
I guess we could do the `sed` here on the output of `black` directly to get the right structure. That is probably cleaner. I'll give it a go 👍

Reply:
Yeah, much simpler, with less installation and so on.